Makefile

```diff
@@ -8,11 +8,17 @@ lint:
 	go get github.com/golang/lint/golint
 	golint *.go
 
-test: sysfs/fixtures/.unpacked
+test: sysfs/fixtures/.copied
```

Comment:
The problem is, the marker needs to be somehow dependent on the files in source code. Otherwise the copy command will never be executed again. That's what I asked @matthiasr to help with, but apparently it's not possible due to Make's lack of colon support in filenames.

I'd prefer slower test execution over stale fixture files. I'd like to prevent developers from unnecessary debug sessions when they forgot to remove the marker file manually to trigger an update.